SLHCUpgradeSimulations/Configuration/python/customise_PFlow.py

```diff
@@ -5,8 +5,8 @@ def customise_extendedTrackerBarrel( process ) :
         for link in process.particleFlowBlock.linkDefinitions:
             if hasattr(link,'trackerEtaBoundary') : link.trackerEtaBoundary = cms.double(3.0)
         for importer in process.particleFlowBlock.elementImporters :
-            if importer.source.value()=="particleFlowClusterHFEM" : importer.importerName = cms.string("ClusterImporterForForwardTracker")
-            if importer.source.value()=="particleFlowClusterHFHAD" : importer.importerName = cms.string("ClusterImporterForForwardTracker")
+            if importer.source.value()=="particleFlowClusterHF" : importer.importerName = cms.string("ClusterImporterForForwardTracker")
```

Comment:
Are we ever going to use the other collections?  Is it worth keeping them in the condition so that if any of the 3 collections are there the change is made?
@bachtis, happy to integrate either way, once tested.